packages/constants/src/category.ts

```diff
@@ -0,0 +1,259 @@
+interface IKey {
+    id: string;
+    label: string;
+    value: string;
+    url: string;
+}
+
+interface ICategory {
+    [category: string]: IKey;
+}
+
+function Key(id: string, label: string, value: string, url: string): IKey {
```

Comment:
i notice the `label` and `value` are always the same, could we do away with one of them and just use the value in both places?

Reply:
updated

Reply:
as discussed, we don't know what `id` is for, but things will break in development if it does affect anything in a serious manner anyways, so we'll be combining this